### PR TITLE
display_banner and prompt_pager line counting/wrapping fix

### PR DESCRIPTION
**Pull request by haliphax**

### CODE REVIEW

File: x84/default/common.py [hunk 1]
```diff
@@ -182,14 +183,18 @@ def show_breaker():
     result = []
     for txt in content:
         if txt.rstrip():
-            result.extend(term.wrap(txt, width, **kwargs))
+            lines = term.wrap(txt, width, **kwargs)
+
+            for line in lines:
```
Comment (signalpillar) [inline x84/default/common.py]: we can `result.extend(lines)`

Comment (jquast) [inline x84/default/common.py]: i could have told you that

       duh

File: CHANGES [hunk 2]
```diff
@@ -1,3 +1,7 @@
+2.0.13
+  - bugfix: common.py's two functions, display_banner and prompt_pager,
+    were not properly counting line numbers. prompt_pager also deals
```
Comment (jquast) [inline CHANGES]: "were not..." phrase is not helpful
"previously, the -more- prompt was not displayed at the correct position" now describes something somebody noticed and may decided is worth upgrading for.

File: x84/default/common.py (hunk 1 above)
Comment (haliphax) [inline x84/default/common.py]: For whatever reason, that was not working properly.

File: CHANGES (hunk 2 above)
Comment (haliphax) [inline CHANGES]: Will update.

**Comment by haliphax** [comment]
gah. attempt to squash fucked the branch history up. closing and re-opening.

**Pull request closed without merging**